cli/wasm/windmill_parser_wasm.generated.js

```diff
@@ -4,9 +4,34 @@
 // deno-fmt-ignore-file
 /// <reference types="./windmill_parser_wasm.generated.d.ts" />
 
-// source-hash: f0b1d0be016a3874b6824c3458a87cf04fc171fd
+// source-hash: 6fa52bb354da6feea9a77d82849885878b16bfa1
 let wasm;
 
+import * as import0 from "env";
```

Comment:
I remember when compiling c# it initially didn't work because the c library expected some dynamically linked libs which couldn't exist in the wasm contexts. These lines were a symptom of that, which makes me believe there might be some errors in the generation, although it might only affect C#.

I'm leaving this comment mainly to myself to go check it, because it might only affect the c# parser